py2pack/__init__.py

```diff
@@ -199,8 +199,8 @@ def _prepare_template_env(template_dir):
 
 def _get_source_url(pypi_name, filename):
     """get the source url"""
-    # example: https://pypi.io/packages/source/u/ujson/ujson-1.2.3.tar.gz
-    return 'https://pypi.io/packages/source/{}/{}/{}'.format(
+    # example: https://files.pythonhosted.org/packages/source/u/ujson/ujson-1.2.3.tar.gz
+    return 'https://files.pythonhosted.org/packages/source/{}/{}/{}'.format(
```

Comment:
Not sure that I like this. Looks like that pypi.io redirect to pypi.org, indeed both redirect to files.pythonhosted.org when are referring to a file.

So I wonder if at the end is not better to change to pypi.org here.